helm/templates/grafana/grafana.deployment.yaml

```
@@ -27,7 +27,7 @@ spec:
       {{- include "capellacollab.pod.spec" . | indent 6 -}}
       containers:
         - name: grafana
-          image: {{ .Values.docker.registry.external -}}/grafana/grafana:9.5.5
+          image: {{ .Values.docker.registry.external }}/grafana/grafana
```

```suggestion
          image: {{ .Values.docker.registry.external }}/grafana/grafana:10.0.1
```

I'd set the version to a specific tag to prevent incompatible versions. I guess the version 9.x.x is not compatible with the changes in the `grafana.configmap.yaml`.